bin/receiver.py

```diff
@@ -249,13 +249,13 @@ def main():
         # The message listening loop.
         while True:
             try:
-                time.sleep(1)
+                time.sleep(0.1)
                 if protocol == Ssm2.AMS_MESSAGING:
                     # We need to pull down messages as part of
                     # this loop when using AMS.
                     ssm.pull_msg_ams()
 
-                if i % REFRESH_DNS == 0:
+                if i % (REFRESH_DNS * 10) == 0:
```

Comment:
Why not just increase the value of REFRESH_DNS by ten?

Reply:
As it's defined as being in seconds and this follows the same pattern used somewhere else. I guess changing the definition might be better, but I was too lazy to look at where `REFRESH_DNS` was defined and to also check that it wasn't being used anywhere else. :stuck_out_tongue_closed_eyes:

Reply:
Will leave as-is.